src/types.js

```diff
@@ -49,6 +49,18 @@ export type InputSelectionType = {
 
 export type Account = Auth;
 
+/**
+ * A list item for all private message list
+ */
+
+export type PmConversationListItem = {
+  ids: string,
+  msgId: number,
+  recipients: string,
+  timestamp: number,
+  unread?: number,
```

Comment:
Why is this optional? We always have it, right?

It looks like Flow agrees: if I remove the `?`, there are no errors.

And that's not because we're not checking, thankfully: if I add to this type definition some nonsense property that shouldn't be there, I get an error in `conversationsSelectors`.